package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "mit-open-login-button",
+  "version": "0.0.1",
+  "description": "A basic login button for your website to log in using MIT Open",
+  "main": "index",
+  "types": "index",
+  "repository": "https://github.com/mitodl/mit-open-login-button",
+  "author": "Carey Gumaer <gumaerc@mit.edu>",
+  "license": "MIT",
+  "scripts": {
+    "lint": "eslint './**/*.{js,jsx,ts,tsx}'",
+    "fmt": "yarn fmt:check --write",
+    "fmt:check": "LOG_LEVEL= prettier-eslint --list-different './**/*.{js,jsx,ts,tsx,scss}'"
```

Comment:
**Strong Suggestion:** Don't use `prettier-eslint`. Just use eslint, and drop a `.pre-commit-config.yaml` file in the repo.